packages/core/src/config/load.ts

```diff
@@ -62,11 +62,15 @@ async function addConfigMetadata({
 }
 
 export async function loadConfig(
-  configPath: string | undefined = findConfig(),
-  customExtends?: string[],
-  processRawConfig?: (rawConfig: RawConfig) => void | Promise<void>
+  options: {
+    configPath?: string;
+    customExtends?: string[];
+    processRawConfig?: (rawConfig: RawConfig) => void | Promise<void>;
+    argv?: ArgsType;
```

Comment:
What do you think of this: 
```suggestion
    files?: string[]
    region?: string
```
?